encord/workflow/stages/consensus_annotation.py

```diff
@@ -40,6 +44,25 @@ def get_tasks(
         dataset_hash: Union[List[UUID], UUID, List[str], str, None] = None,
         data_title: Optional[str] = None,
     ) -> Iterable[ConsensusAnnotationTask]:
+        """
+        **Params**
+
+        - assignee: User assigned to a task.
+        - data_hash: Unique ID for the data unit.
+        - dataset_hash: Unique ID for the dataset that the data unit belongs to.
+        - data_title: Name of the data unit.
+
+        **Returns**
+
+        Returns tasks in the stage with the following information:
+
+        - uuid: Unique identifier for the task.
+        - created_at: Time and date the task was created.
+        - updated_at: Time and date the task was last edited.
+        - data_hash: Unique identifier for the data unit.
+        - data_title: Name/title of the data unit.
+        - subtasks: A list of subtasks that follow the task format for AnotationTask.
```

Comment:
I'd rather mention that this method returns a list of `ConsensusAnnotationTask` classes. And document fields as part of the `ConsensusAnnotationTask` class.

Reply:
Updated